lmdeploy/lite/apis/calibrate.py

```diff
@@ -262,7 +262,13 @@ def calibrate(model: str,
         if dtype == 'float16':
             model.half()
         elif dtype == 'bfloat16':
+            assert torch.cuda.is_bf16_supported(
+            ), 'your device does not support bfloat16 please set --dtype float16'  # noqa
             model.to(torch.bfloat16)
+        elif dtype == 'auto' and model.config.torch_dtype == torch.bfloat16:
```

Comment:
Is it possible that `model.config.torch_dtype` not exist?

Reply:
Since v4.9.0, transformers `PretrainedConfig` contains `torch_dtype` attribute.